scripts/pi-hole/php/settings_dhcp.php

```diff
@@ -0,0 +1,279 @@
+                            <h3 class="box-title">Advanced DHCP settings</h3>
+                            <div class="box-tools pull-right"><button type="button" class="btn btn-box-tool" data-widget="collapse"><i class="fa fa-plus"></i></button></div>
+                        </div>
+                        <div class="box-body">
+                            <div class="col-md-12">
+                                <div class="form-group">
+                                    <div class="checkbox"><label><input type="checkbox" name="useIPv6" <?php if($DHCPIPv6){ ?>checked<?php } ?> class="DHCPgroup" <?php if(!$DHCP){ ?>disabled<?php } ?>> Enable IPv6 support (SLAAC + RA)</label></div>
+                                </div>
+                            </div>
+                            <div class="col-md-12">
+                            <label>Pi-hole domain name</label>
+                            <div class="form-group">
+                                <div class="input-group">
+                                    <div class="input-group-addon">Domain</div>
+                                        <input type="text" class="form-control DHCPgroup" name="domain" value="<?php echo $piHoleDomain; ?>" <?php if(!$DHCP){ ?>disabled<?php } ?>>
+                                </div>
+                            </div>
+                            </div>
+                            <div class="col-md-12">
+                            <label>DHCP lease time</label>
+                            <div class="form-group">
+                                <div class="input-group">
+                                    <div class="input-group-addon">Lease time in hours</div>
+                                        <input type="text" class="form-control DHCPgroup" name="leasetime" id="leasetime" value="<?php echo $DHCPleasetime; ?>" data-inputmask="'mask': '9', 'repeat': 7, 'greedy' : false" data-mask <?php if(!$DHCP){ ?>disabled<?php } ?>>
+                                </div>
+                            </div>
+                                <p>Hint: 0 = infinite, 24 = one day, 168 = one week, 744 = one month, 8760 = one year</p>
+                            </div>
+                        </div>
+                    </div>
+                    </div>
+<?php
+$dhcp_leases = array();
+if($DHCP) {
+    // Read leases file
+    $leasesfile = true;
+    $dhcpleases = @fopen('/etc/pihole/dhcp.leases', 'r');
+    if(!is_resource($dhcpleases))
+        $leasesfile = false;
+
+    function convertseconds($argument) {
+        $seconds = round($argument);
+        if($seconds < 60)
+        {
+            return sprintf('%ds', $seconds);
+        }
+        elseif($seconds < 3600)
+        {
+            return sprintf('%dm %ds', ($seconds/60), ($seconds%60));
+        }
+        elseif($seconds < 86400)
+        {
+            return sprintf('%dh %dm %ds',  ($seconds/3600%24),($seconds/60%60), ($seconds%60));
+        }
+        else
+        {
```

Comment:
This from the original code in the DHCP block of settings.php. 

@DL6ER / @Mcat12: Would you like me to remove the `else`? I personally prefer it, but @codacy-bot seems to be of a different opinion...